src/utils/userModeling.js

```
@@ -10,9 +10,11 @@ const cinemaUserModeling = async (cinemas, username) => {
 
   if (userReservations.length) {
     let cinemaResult = {};
-    userReservations.map(userReservation => {
+    userReservations.map((userReservation) => {
       const id = userReservation.cinemaId;
-      cinemaResult.hasOwnProperty(id) ? ++cinemaResult[id] : (cinemaResult[id] = 1);
+      cinemaResult.hasOwnProperty(id)
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)